ee/codegen/src/generators/workflow.ts

```diff
@@ -184,7 +184,11 @@ export class Workflow {
     });
     workflowDisplayClass.inheritReferences(workflowClassRef);
 
-    const entrypointNode = this.workflowContext.getEntrypointNode();
+    const entrypointNode = this.workflowContext.tryGetEntrypointNode();
+    if (!entrypointNode) {
+      return workflowDisplayClass;
+    }
```

Comment:
Don't early return. Instead conditionally filter out entrypoint related arguments from the `WorkflowMetaDisplay()` instance